src/styles/blocks/head-block/header.scss

```diff
@@ -0,0 +1,31 @@
+.header {
+  position: relative;
+  display: flex;
+  justify-content: flex-end;
+  max-width: 1200px;
+  padding: 0 50px;
+  height: 60px;
```

Comment:
Consider avoiding setting heights for blocks, the block should have height inherited from elements inside